pkg/cmd/pr/diff/diff.go

```diff
@@ -278,3 +286,56 @@ func changedFilesNames(w io.Writer, r io.Reader) error {
 
 	return nil
 }
+
+func sanitizedReader(r io.Reader) io.Reader {
+	return transform.NewReader(r, sanitizer{})
+}
+
+// sanitizer replaces non-printable characters with their printable representations
+type sanitizer struct{ transform.NopResetter }
+
+// Transform implements transform.Transformer.
```

Comment:
TIL about `transform.Transformer`. I would like to change our other ASCII sanitization code to use this interface.